### PR TITLE
Add calculator structure with Components

### PR DESCRIPTION
**Pull request by SaboorSohaib**
### The following requirements are applied.

- Create a directory called components.
- Inside components create a new Calculator.js file.
- In Calculator.js, create a React component that matches the design.
- Use class-based components (All components should be class-based).

**Kindly check my pull request if there is something wrong.**

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,29 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/calculator';
 
 function App() {
   return (
```
Comment (Mihndim2020) [inline src/App.js]: - [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)for this project, " `Use class based components (All components should be class-based).` ", please kindly change your `App.js` component which a currently a functional component to a class-based component.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @SaboorSohaib, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION



# Status: Requires Changes ♻️
_Check the comments under the review._

- [ ] As it is a [requirement ](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md)for this project, " `Delete all the boilerplate from CRA (text, images, styles).` ", please kindly delete all files that are unrelated to the project, as this will make your repository less crowded. 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (SaboorSohaib) [reply src/App.js]: Hi @Mihndim2020 

Shall I delete the files which are in the Public folder or there is something else?

**Review by mikethreels — APPROVED** [review]
Hi Sohaib, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- linters are passing
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**